File: The Core/Intro Gates/8. Phone Call/phoneCall.py
def phoneCall(min1, min2_10, min11, s):
    centsRemaining = s
    minutes = 0
    
    if min1 <= centsRemaining:
        centsRemaining -= min1
        minutes += 1
    else:
        return minutes;

    for x in range(1, 10):
        if min2_10 < centsRemaining:
            centsRemaining -= min2_10
            minutes += 1
        else:
            return minutes
        
    rem = centsRemaining / min11
    minutes += int(math.floor(rem))
    
    return minutes

File: The Core/Intro Gates/8. Phone Call/phoneCall.lua
function phoneCall(min1, min2_10, min11, s)
    centsRemaining = s
    minutes = 0
    
    if min1 <= centsRemaining then
        centsRemaining = centsRemaining - min1
        minutes = minutes + 1
    else
        return minutes;
    end 
    
    for x = 3, 11 do
        if min2_10 < centsRemaining then
            centsRemaining = centsRemaining - min2_10
            minutes = minutes + 1
        else
            return minutes
        end
    end
    
    rem = centsRemaining / min11
    minutes = minutes + math.floor(rem)
    
    return minutes
end
